services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -248,6 +251,25 @@ def get_dynamic_sidecar_spec(
                     "uuid": f"{scheduler_data.node_uuid}",
                 },
                 "Mounts": mounts,
+                "Secrets": [
+                    {
+                        "SecretID": app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID,
+                        "SecretName": app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME,
+                        "File": {
+                            "Name": app_settings.DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME,
+                            "Mode": 444,
+                            "UID": "0",
+                            "GID": "0",
```

Comment:
Question about this `UID` and `GID`. Could they be different from 0? I'd guess 0 is root right?

Reply:
any comments on this?